desktop/ui/src/main/resources/datacleaner-home/jobs/OrderDB Name and Phone Cleansing.analysis.xml

```diff
@@ -0,0 +1,200 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<job xmlns="http://eobjects.org/analyzerbeans/job/1.0">
+	<job-metadata>
+		<job-description>Example job that cleanses the names, phone in the
+			database
+		</job-description>
+		<metadata-properties>
+			<property name="CoordinatesY.PUBLIC.CUSTOMERS">93</property>
+			<property name="CoordinatesX.PUBLIC.CUSTOMERS">62</property>
+			<property name="DemoJob">true</property>
+		</metadata-properties>
+	</job-metadata>
+	<source>
+		<data-context ref="orderdb" />
+		<columns>
+			<column id="col_customernumber" path="CUSTOMERS.CUSTOMERNUMBER"
+				type="INTEGER" />
+			<column id="col_customername" path="CUSTOMERS.CUSTOMERNAME"
+				type="VARCHAR" />
+			<column id="col_contactlastname" path="CUSTOMERS.CONTACTLASTNAME"
+				type="VARCHAR" />
+			<column id="col_contactfirstname" path="CUSTOMERS.CONTACTFIRSTNAME"
+				type="VARCHAR" />
+			<column id="col_phone" path="CUSTOMERS.PHONE" type="VARCHAR" />
+			<column id="col_addressline1" path="CUSTOMERS.ADDRESSLINE1"
+				type="VARCHAR" />
+			<column id="col_addressline2" path="CUSTOMERS.ADDRESSLINE2"
+				type="VARCHAR" />
+			<column id="col_city" path="CUSTOMERS.CITY" type="VARCHAR" />
+			<column id="col_state" path="CUSTOMERS.STATE" type="VARCHAR" />
+			<column id="col_postalcode" path="CUSTOMERS.POSTALCODE" type="VARCHAR" />
+			<column id="col_country" path="CUSTOMERS.COUNTRY" type="VARCHAR" />
+			<column id="col_salesrepemployeenumber" path="CUSTOMERS.SALESREPEMPLOYEENUMBER"
+				type="INTEGER" />
+			<column id="col_creditlimit" path="CUSTOMERS.CREDITLIMIT"
+				type="NUMERIC" />
+			<column id="col_jobtitle" path="CUSTOMERS.JOBTITLE" type="VARCHAR" />
+		</columns>
+	</source>
+	<transformation>
+		<transformer>
+			<descriptor ref="Name Correction" />
+			<metadata-properties>
+				<property name="CoordinatesY">87</property>
+				<property name="CoordinatesX">236</property>
+			</metadata-properties>
+			<properties>
+				<property name="Connection" value="named:demo" />
```

Comment:
Check the common-datacleaner project. I think it's just "demo" but I cannot recall. It must be in the Converter class that handles ConnectionInformation conversion